tools/db_crashtest.py

```diff
@@ -137,6 +137,7 @@
     "index_block_restart_interval": lambda: random.choice(range(1, 16)),
     "use_multiget": lambda: random.randint(0, 1),
     "periodic_compaction_seconds": lambda: random.choice([0, 0, 1, 2, 10, 100, 1000]),
+    "stats_dump_period_sec": lambda: random.choice([0, 30]),
```

Comment:
Is there any reason to set it to 0 (disable)? Maybe use the default of 600 instead of 0.

Reply:
I know we have at least one internal user setting to 0. Hypothetically, if stats dump does some kind of reset on some data that prevents some overflow when run periodically, it could hide a bug. That's hypothetical but I think sufficiently motivates 0. Perhaps I'll do [0, 10, 600].